src/compas_fab/utilities/file_io.py

```diff
@@ -32,13 +43,30 @@ def read_csv_to_dictionary(csvfile, delimiter=';'):
 
 def write_data_to_json(data, file):
     """Write data to json file.
+
+    Parameters
+    ----------
+    data : dict, list, tuple, str, unicode, int, long, float, boolean, None
+        The data to write to json file. Data must be JSON serialisable.
+    file : str
+        TThe path where to save the data.
```

Comment:
Should you want to work on your stutter, I have a few speech therapist friends....

Reply:
haha!